homeassistant/components/tellduslive/sensor.py

```diff
@@ -194,4 +194,4 @@ def native_value(self):
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
-        return "-".join(self._id)
+        return "-".join(map(str, self._id))
```

Comment:
We should do a migration for the unique_ids that are an integer, otherwise it will create duplicate entities

Reply:
Example for an unique_id migration: https://github.com/home-assistant/core/pull/122330

Reply:
But this way can't produce an int as unique_id, unless `self._id` is not a list

Reply:
No we want it to be a string, but what I am wondering about, this can only return an integer when `self._id` only returns a single value. So I think the change itself is doing something wrong. When you run this code, do you see what the unique_id of the entity ends up being?

Reply:
Oh I think I see what you mean, this looks good